biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
```

Comment:
I like it. How about adding "living" to ecosystem?
Biotope is a framework which enables you to develop scalabale, platform-agnostic and living frontend component ecosystems
And when you wonder what a living ecosystem is? It's alive, growing, breathing, moving...